### PR TITLE
Features/error source generator

### PR DESCRIPTION
**Pull request by luizfbicalho**
Closes: #4160

### CODE REVIEW

File: Source/Csla.Generators/cs/AutoImplementProperties/Csla.Generator.Tests/Snapshots/GeneratorSnapshotTests.ReadOnlyBaseInterface#InterfaceImplementBusinessPOCO.g.verified.cs [hunk 1]
```diff
@@ -3,6 +3,7 @@
 #nullable enable
 using System;
```
Comment (rockfordlhotka) [inline Source/Csla.Generators/cs/AutoImplementProperties/Csla.Generator.Tests/Snapshots/GeneratorSnapshotTests.ReadOnlyBaseInterface#InterfaceImplementBusinessPOCO.g.verified.cs:4]: Should this file be in source control? It is generated?

**Review by rockfordlhotka — CHANGES_REQUESTED** [review]

File: Source/Csla.Generators/cs/AutoImplementProperties/Csla.Generator.Tests/Snapshots/GeneratorSnapshotTests.ReadOnlyBaseInterface#InterfaceImplementBusinessPOCO.g.verified.cs (hunk 1 above)
Comment (luizfbicalho) [reply Source/Csla.Generators/cs/AutoImplementProperties/Csla.Generator.Tests/Snapshots/GeneratorSnapshotTests.ReadOnlyBaseInterface#InterfaceImplementBusinessPOCO.g.verified.cs:4]: in this video https://www.youtube.com/watch?v=azJm_Y2nbAI the author says that is a good practice to put the nullable enable in teh begining

Comment (rockfordlhotka) [reply Source/Csla.Generators/cs/AutoImplementProperties/Csla.Generator.Tests/Snapshots/GeneratorSnapshotTests.ReadOnlyBaseInterface#InterfaceImplementBusinessPOCO.g.verified.cs:4]: I agree with the `#nullable enable` - I am wondering if the `.g.verified.cs` files should be in source control at all - if they are generated on each build they should not be in source control.

Comment (luizfbicalho) [reply Source/Csla.Generators/cs/AutoImplementProperties/Csla.Generator.Tests/Snapshots/GeneratorSnapshotTests.ReadOnlyBaseInterface#InterfaceImplementBusinessPOCO.g.verified.cs:4]: this is part of the snapshot unit test, when the test runs it makes a .received.cs and compares both files

the verivied is there to check the test result

**Review by rockfordlhotka — APPROVED** [review]

**Pull request merged**